scripts/.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "../.eslintrc.js",
   "rules": {
-    "no-console": "off"
+    "no-console": "off",
+    "import/no-extraneous-dependencies": "off"
```

Comment:
**question:** This rule is already configured at the root of the monorepo but affect only devDeps and peerDeps. Couldn't the missing dependency be added in order to prevent turning off this rule?